Tuto-GUDHI-Barycenters-of-persistence-diagrams.ipynb

```diff
@@ -376,7 +376,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.3"
+   "version": "3.8.2"
```

Comment:
Did you mean to touch this file?

Reply:
Hum no actually; I guess I just opened the notebook at some point and didn't notice something was changed.